packages/framework/src/Model/Feed/FeedXmlWriter.php

```diff
@@ -68,18 +69,15 @@ public function writeItems(array $items, DomainConfig $domainConfig, $feedTempla
     }
 
     /**
-     * @param \Twig_Template $twigTemplate
+     * @param \Twig_TemplateWrapper $twigTemplate
      * @param string $name
      * @param array $parameters
      * @return string
      */
-    private function getRenderedBlock(Twig_Template $twigTemplate, $name, array $parameters = [])
+    private function getRenderedBlock(Twig_TemplateWrapper $twigTemplate, $name, array $parameters = [])
```

Comment:
it would be nice to rename `$twigTemplate` to `$twigTemplateWrapper`

Reply:
@vitek-rostislav ...or you can cover these situations by a fixer: https://github.com/symplify/codingstandard#property-name-should-match-its-type-if-possible